### ISSUE TITLE
Returning to different study in results list

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cdc.versions#172]
**Original report on BitBucket by Taina Jääskeläinen.**

----------------------------------------

1. The user clicks on a study in the results list
2. He reviews the study and clicks on the back arrow on top to return to the result list
3. He does not return to the study in the result list where he was but to the top of that page of the list.

This makes it difficult for the user to remember where he was in the results list, particularly if they were towards the bottom of the page in the list. Functional navigation issue.

‌

**Comment by MortenSikt** [comment]
Markus take a quick look if there is an easy fix, if not wait for fix in coordinate

**Comment by MortenSikt** [comment]
Have you had the chance to look at this @markusjt ?

**Comment by markusjt** [comment]
Yes and there are ways to accomplish it but nothing simple. I'd say it is not worth the effort, if we can change to the upgraded thematic view codebase in the next few months, which should be fine. This can probably be removed from milestone and closed unless we want to discuss further in sprint.

**Comment by markusjt** [comment]
Yes, although I've recently noticed that enabling autofocus for search field breaks it so either autofocus needs to be always disabled or need to make it so that it's only disabled when going back in history. First solution is very simple since it's just changing the autofocus attribute value for search field but second would basically let us have both at the same time.

**Comment by MortenSikt** [comment]
Ok so autofocus is what brings you to the search box then when going back? But what is the benefit of having it in the first place (might be a stupid question, i have not heard of this before)?

**Comment by markusjt** [comment]
Exactly, autofocus allows user to immediately start typing search term on search page since it will be focused. Personally I don't really care for autofocus but I guess it makes sense when one element on the page is usually a "clear starting point".
